README.md

```diff
@@ -106,9 +106,17 @@ This tag contains more advance setup tasks, such as:
 - Install supplementary packages - not just sb_debian_base_extra_packages
 - Enable firewall using UFW
     - Open general ports (e.g. SSH port, HTTP port; by default SSH)
-        - You can define {{ ports }}
+        - You can define {{ sb_debian_base_ports }}
     - Open specific ports for specific IPs
-        - You can define {{ port_ips }}
+        - You can define {{ sb_debian_base_ports_ips }} like:
+          ```yml
+          sb_debian_base_ports-ips:
```

Comment:
@speedlight I think you miss a `_` here:

```diff
- sb_debian_base_ports-ips
+ sb_debian_base_ports_ips
```

Reply:
Thanks!